flutter/lib/src/default_integrations.dart

```diff
@@ -8,6 +8,40 @@ import 'package:sentry/sentry.dart';
 import 'sentry_flutter_options.dart';
 import 'widgets_binding_observer.dart';
 
+class RunZoneGuardedIntegration extends Integration<SentryFlutterOptions> {
+  RunZoneGuardedIntegration(this._runBeforeIntegrations, this._runAfterIntegrations);
+
+  final Function() _runBeforeIntegrations;
+  final Function() _runAfterIntegrations;
+
+  @override
+  FutureOr<void> call(Hub hub, SentryFlutterOptions options) {
+    runZonedGuarded(() async {
```

Comment:
```suggestion
  @override
  FutureOr<void> call(Hub hub, SentryFlutterOptions options) async {
    await runZonedGuarded(() async {
```

Reply:
I checked this again and was not able to call `await` on `runZonedGuarded `, as we are running in a timeout in the case test case where a error is thrown. See `default_integrations_test.dart` 'Run zoned guarded calls catches integrations errors'.